### ISSUE TITLE
refactor helpers.js

### ISSUE THREAD

**Issue by tabrindle** [tabrindle/envinfo#71]
helpers.js is a bit large and unwieldy, and it does need refactored in the future. 

I think to keep file bloat down and a similar organization to the output of envinfo, we should organize the helpers like this

```
helpers/
  binaries.js
  browsers.js
  databases.js
  IDEs.js
  index.js
  languages.js
  SDKs.js
  servers.js
  system.js
  utilities.js
  virtualization.js
```

@gengjiawen

**Comment by gengjiawen** [comment]
This looks fantastic :)

**Comment by tabrindle** [comment]
fixed by #89
